lib/index-static-watcher.js

```diff
@@ -0,0 +1,120 @@
+// This is essentially a webpack-based compiler for
+// index-static.jsx. Much of the code is largely
+// based on James Long's "Backend Apps With Webpack"
+// series of blog posts:
+//
+// http://jlongster.com/Backend-Apps-with-Webpack--Part-I
+//
+// Doing this allows us to use the same build toolchain
+// for node and the browser, and also increases
+// responsiveness during development via hot-loading.
+
+var fs = require('fs');
+var path = require('path');
+var webpack = require('webpack');
+
+function IndexStaticWatcher(options) {
+  var self = {};
+  var outputDir = options.outputDir;
+  var outputFilename = 'index-static.bundle.js';
+  var indexStatic = null;
+  var buildInProgress = false;
+  var entry = path.join(__dirname, 'index-static.jsx');
+  var compiler = webpack({
+    entry: entry,
+    target: 'node',
+    devtool: 'sourcemap',
+    externals: function(context, request, callback) {
```

Comment:
Ah yeah, this was annoying for me to figure out, because webpack's documentation is sparse in some places. :( I think I literally just `console.log`'d the arguments to get an idea of what they actually were, lol. Not a very strong sign of forwards-compatibility, but the good news is that at least we'll know when it breaks, since that code is executed so often!